tests/custom_lookups/tests.py

```diff
@@ -63,6 +63,15 @@ def as_sql(self, compiler, connection):
         return '3 * (%s)' % lhs, lhs_params
 
 
+class LastDigitBilateralTransform(models.Transform):
+    bilateral = True
```

Comment:
It doesn't, I only did it to match with `mul3` and `div3`, as that seemed the closest match in the test suite for what I was trying to demonstrate.